exercises/sum.cpp

```diff
@@ -6,3 +6,16 @@
   Insert the second number: 2
   Sum: 3
 */
+
+#include <iostream>
+
+using namespace std;
+
+int main(){
+  int x, y;
+  cout << "Insert the first number: ";
+  cin >> x;
+  cout << "\nInsert the second number: ";
+  cin >> y;
+  cout << "\nSum: " << x+y << endl;
+}
```

Comment:
manca return 0, inoltre potresti sostituire 0 con EXIT_SUCCESS in modo da migliorare la leggibilità del codice
```c++
#define EXIT_SUCCESS 0
...
return EXIT_SUCCESS;
```